include/emqx_ct_jobs_transform.hrl

```diff
@@ -14,6 +14,9 @@
 %% limitations under the License.
 %%--------------------------------------------------------------------
 
+-ifndef(EMQX_CT_JOBS_TRANSFORM).
+-define(EMQX_CT_JOBS_TRANSFORM, true).
```

Comment:
Whats the intent with the EMQX_CT_JOBS_TRANSFORM  ?
A test suite using jobs matrix, adds -compile([{parse_transform, emqx_ct_jobs_suite_transform}]).
and thus no need for a flag ?

Reply:
Should have been `EMQX_CT_JOBS_TRANSFORM_HRL`